lib/good_job.rb

```diff
@@ -37,14 +37,30 @@ module GoodJob
   #   @return [Boolean]
   mattr_accessor :preserve_job_records, default: false
 
-  # @!attribute [rw] reperform_jobs_on_standard_error
+  # @!attribute [rw] retry_on_unhandled_error
   #   @!scope class
   #   Whether to re-perform a job when a type of +StandardError+ is raised to GoodJob (default: +true+).
   #   If +true+, causes jobs to be re-queued and retried if they raise an instance of +StandardError+.
   #   If +false+, jobs will be discarded or marked as finished if they raise an instance of +StandardError+.
   #   Instances of +Exception+, like +SIGINT+, will *always* be retried, regardless of this attribute's value.
   #   @return [Boolean]
-  mattr_accessor :reperform_jobs_on_standard_error, default: true
+  mattr_accessor :retry_on_unhandled_error, default: true
+
+  # Deprecated
```

Comment:
Do you know a proper way of documenting deprecations in yard?

Reply:
`@depreacted` is the tag to use:

```suggestion
  # @deprecated Use {GoodJob#retry_on_unhandled_error} instead.
```

![Screen Shot 2020-10-01 at 8 50 53 AM](https://user-images.githubusercontent.com/47554/94832811-3bb29f00-03c3-11eb-97f0-3d00170b981b.png)

Reply:
Thanks, updated.